torch/onnx/operators.py

```diff
@@ -0,0 +1,24 @@
+import torch
+from torch.autograd import Variable
+import torch.onnx
+
+# This file contains pure pytorch operators that should eventually
+# find a home outside of the onnx/ directory
+
+
+def _size_as_variable(g, input):
+    return g.op('Shape', input)
+
+
+@torch.onnx.symbolic_override(_size_as_variable)
+def size_as_variable(var):
+    return Variable(torch.LongTensor(tuple(var.size())))
```

Comment:
We've merged Variable and Tensor so the `Variable` call is no longer necessary and `size_as_tensor`  (or `shape_as_tensor`) might be a better name.

Reply:
++